src/hyperion/experiment_plans/wait_for_robot_load_then_centre.py

```diff
@@ -27,15 +34,28 @@ def create_devices(context: BlueskyContext) -> GridDetectThenXRayCentreComposite
     return device_composite_from_context(context, GridDetectThenXRayCentreComposite)
 
 
-def wait_for_robot_load_then_centre(
+def wait_for_smargon_not_disabled(smargon: Smargon, timeout=60):
+    LOGGER.info("Waiting for smargon enabled")
+    SLEEP_PER_CHECK = 0.1
+    times_to_check = int(timeout / SLEEP_PER_CHECK)
+    for _ in range(times_to_check):
+        smargon_disabled = yield from bps.rd(smargon.disabled)
+        if not smargon_disabled:
+            LOGGER.info("Smargon now enabled")
+            return
+        yield from bps.sleep(SLEEP_PER_CHECK)
+    raise TimeoutError(
+        "Timed out waiting for smargon to become enabled after robot load"
+    )
+
+
+def wait_for_robot_load_then_centre_plan(
     composite: GridDetectThenXRayCentreComposite,
     parameters: WaitForRobotLoadThenCentreInternalParameters,
-) -> MsgGenerator:
-    # Start arming the detector
-
+):
     # Move backlight in
```

Comment:
Are these comments things the plan might need to do in a future PR?

Reply:
Yes, I agree that they probably shouldn't really be in the codebase, I'll remove them